test/test_time_integrators.py

```diff
@@ -68,3 +77,60 @@ def rhs(t, state):
 
     logger.info(f"Time Integrator EOC:\n = {integrator_eoc}")
     assert integrator_eoc.order_estimate() >= method_order - .01
+
+
+@pytest.mark.parametrize(("method", "method_order"), [
+    (ODE23MethodBuilder("y", use_high_order=False), 2),
+    (ODE23MethodBuilder("y", use_high_order=True), 3),
+    (ODE45MethodBuilder("y", use_high_order=False), 4),
+    (ODE45MethodBuilder("y", use_high_order=True), 5),
+    (ForwardEulerMethodBuilder("y"), 1),
+    (MidpointMethodBuilder("y"), 2),
+    (HeunsMethodBuilder("y"), 2),
+    (RK3MethodBuilder("y"), 3),
+    (RK4MethodBuilder("y"), 4),
+    (RK5MethodBuilder("y"), 5),
+    (LSRK4MethodBuilder("y"), 4),
+    (KennedyCarpenterIMEXARK4MethodBuilder("y", use_implicit=False,
+        explicit_rhs_name="y"), 4),
+    (SSPRK22MethodBuilder("y"), 2),
+    (SSPRK33MethodBuilder("y"), 3),
+    ])
```

Comment:
That's a lot of methods... I suspect this would eat a lot of CI time. We should probably cut this down some. (Could you report the timings of how much this adds?)

Reply:
<img width="933" alt="Screen Shot 2021-04-21 at 1 45 32 PM" src="https://user-images.githubusercontent.com/25389326/115624862-d40d6400-a2c0-11eb-8aef-3331a77ff0f1.png">
 This is on porter, where running all of the tests takes about 20 minutes. I don't see this as a CI issue, but nevertheless, I can pare down the methods tested as requested.